ajax/libs/backbone.stickit/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "backbone.stickit",
-  "description": "Model binding in Backbone style.",
   "filename": "backbone.stickit.min.js",
+  "description": "Model binding in Backbone style.",
```

Comment:
Let's make it back.

Reasons:
1. It's not necessary.
2. You don't even mention this change in the commit message.